gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Unreleased Changes
 ------------------
 
+3.125.6 (2022-01-31)
```

Comment:
This should just be an entry under "Unreleased changed" - and lets mark this as a feature (which means it will get a minor version bump rather than a patch version).  The SDK's release system will automatically take changes from the "Unreleased changes" section, update versions and publish new gem versions.